examples/custom-tests/README.md

```diff
@@ -17,7 +17,7 @@ test:
           - <paths glob>
 ```
 
-As tests take time, you might prefer to configure tests using [profiles](https://skaffold.dev/docs/environment/profiles/) so that they can be automatically enabled or disabled, e.g.
+As tests take time, you might prefer to configure tests using [profiles](https://skaffold.dev/docs/https://skaffold.dev/docs/environment/profiles/) so that they can be automatically enabled or disabled, e.g.
```

Comment:
This change doesn't look right

Reply:
no it sure doesn't, this was autogenerated by our release script...

Reply:
this slipped through in https://github.com/GoogleContainerTools/skaffold/pull/5451, but i'm really not sure how it's just getting synced now. fixed anyway